manager/users/models.py

```diff
@@ -179,6 +179,26 @@ def get_feature_flags(user: User) -> Dict[str, str]:
     return features
 
 
+def get_custom_attributes(user: User) -> Dict[str, Union[bool, int, str]]:
+    """
+    Get a dictionary of custom attributes for the user.
+
+    These are summary attributes of the user, derived from other data,
+    that do not fit into one of the other groups e.g. `get_projects_summary`.
+    They are added as needed, principally for use in external integrations.
+    """
+    # User is a member of at least one project that is (a) owned by eLife, or
+    # (b) has an eLife source.
+    elife_author = (
+        get_projects(user, include_public=False)
+        .filter(Q(account__name="elife") | Q(sources__address__startswith="elife://"))
+        .count()
```

Comment:
I think for the purposes of user segmenting, the second scenario of any user with a project with an eLife source is an unnecessary criteria.

```suggestion
    # User is a member of at least one project that is (a) owned by eLife.
    elife_author = (
        get_projects(user, include_public=False)
        .filter(Q(account__name="elife"))
        .count()
```